tests/test_cot_verify.py

```diff
@@ -654,23 +653,20 @@ def test_find_sorted_task_dependencies(task, expected, task_type):
 @pytest.mark.asyncio
 async def test_build_task_dependencies(chain, mocker):
     async def fake_task(task_id):
-        if task_id == "die":
-            raise TaskclusterFailure("dying")
```

Comment:
Nice cleanup! I see `build_task_dependencies` doesn't raise `TaskclusterFailure` (anymore?)

Reply:
I think this was to get coverage in the try/except that I removed above. Now that we're throwing `CoTError` on a bad `queue.task()` result instead of `TaskclusterFailure`, we don't have to catch `TaskclusterFailure` and make sure we're throwing `CoTError`.

Reply:
Looks like I needed to keep this test code, but change the exception to `CoTError`, for full coverage.